packages/sane/example/main.dart

```diff
@@ -3,9 +3,15 @@
 import 'dart:io';
 import 'dart:typed_data';
 
+import 'package:logging/logging.dart';
 import 'package:sane/sane.dart';
 
 void main(List<String> args) async {
+  Logger.root.level = Level.ALL;
+  Logger.root.onRecord.listen((record) {
+    print('${record.level.name}: ${record.time}: ${record.message}');
```

Comment:
I tested it locally, and nothing shown in the debug console 🤔

Reply:
I actually cannot test it, but you're surly right.

Reply:
@Craftplacer I found a way to redirect log messages to the main thread, let's merge your PR and then I push my changes :)